plugins/processors/noise/README.md

```diff
@@ -0,0 +1,37 @@
+# Noise Processor
+
+The *Noise* processor is used to add noise to numerical field values. For each field a noise is generated using Laplace distribution and added to the value.
+Using the _sensitivity_ and _epsilon_ parameter the effect of the noise can be modified. A bigger _epsilon_ (compared to a smaller _sensitivity_) value means that noise variation will be smaller and the the resulting value will be more close to the real value.
+
+### Configuration
+```toml
+[[processors.noise]]
+  sensitivity = 1.0
+  epsilon = 1.0
+  ignore_fields = []
+  ignore_measurements = []
+```
+
+### Example
+Add noise to each value the *Inputs.CPU*  plugin generates, except for the _uptime\_format_ field and all fields of the metrics _swap_, _disk_ and _net_:
+
+```toml
+[[inputs.cpu]]
+  percpu = true
+  totalcpu = true
+  collect_cpu_time = false
+  report_active = false
+
+[[processors.noise]]
+  sensitivity = 1.0
+  epsilon = 1.0
+  ignore_fields = ["uptime_format"]
+  ignore_measurements = ["swap", "disk", "net"]
```

Comment:
This can be achieved using telegraf's `namepass`/`namedrop` [config options](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#selectors). No need to add this option then.

Reply:
Implemented with 99eb43507214d0e36054cce7c88a8540c54d1210